tests/test_graphical_units/test_img_and_svg.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from pathlib import Path
+
+import pytest
+from manim import *
+from ..helpers.path_utils import get_project_root
+from ..utils.testing_utils import get_scenes_to_test
+from ..utils.GraphicalUnitTester import GraphicalUnitTester
+
+
+class SVGMobjectTest(Scene):
+    def construct(self):
+        path = (
+            get_project_root()
+            / "tests/test_graphical_units/img_svg_resources/weight.svg"
+        )
+        svg_obj = SVGMobject(str(path))
+        self.add(svg_obj)
+        self.wait()
+
+
+class ImageMobjectTest(Scene):
+    def construct(self):
+        file_path = (
+            get_project_root()
+            / "tests/test_graphical_units/img_svg_resources/tree_img_640x351.png"
+        )
+        im1 = ImageMobject(file_path).shift(4 * LEFT + UP)
+        im2 = ImageMobject(file_path, scale_to_resolution=1080).shift(
+            4 * LEFT + 2 * DOWN
+        )
+        im3 = ImageMobject(file_path, scale_to_resolution=540).shift(4 * RIGHT)
+        self.add(im1, im2, im3)
+        self.wait(1)
+
+
+MODULE_NAME = "img_and_svg"
```

Comment:
This module name violates the convention mentioned in https://github.com/ManimCommunity/manim/wiki/Testing#graphical-unit-test that this should be named like a module library.

Personally, I am fine with being more flexible here, but I'd like to make sure that @ManimCommunity/tests is okay with this.

Reply:
Okay.
As these would be only files with each containing only one test I would like to keep them together.
I can make  a note for this in the beginning of the file

Reply:
I don't see a problem with having two one-test files: We currently also have `conftest.py` and `test_axes.py` both containing just one test file. Personally, I don't see a reason to make an exception here, but it is certainly possible that I am reading the guidelines too strict.

But anyhow, this is why I would like to from @ManimCommunity/tests: how strict are the naming conventions in the testing guidelines? Should this be split to `test_image_mobject.py` and `test_svg_mobject.py`, or is it okay to leave this as-is?

Reply:
It looks fine to me, as `MODULE_NAME` refers to the name of the directory where the control_data will be stored, and not to a module as python calls it.

What is wrong with it ?

Reply:
The linked guidelines explicitly state "Follow the structure presented above: the filename must be something like test_<module_name>.py.". I know that it is not *required*, but the current guidelines say that the file **must** be named as `test_<module_name>.py`, which is not the case here.

I am asking because I don't know how strict **must** should be read.

Reply:
But, the test file is named `test_img_and_svg.py`, so it's fine, no ?

Reply:
It seems I read it differently as it is meant. I understand this note in the sense that `<module_name>` should be the name of a library module.

As this does not seem to be the case, I am fine with as it is.

Reply:
Actually, I find the wording in the guidelines confusing: there is no `img_and_svg` module anywhere, is there? But in any case, this is nothing that has to be resolved or discussed here.